### PR TITLE
chore (refs T36840): add missing Hooks/Selectors.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://demoseurope.youtrack.cloud/issue/DPP-2746/Missing-hooks-Filterung-nach-Einreichung

- add missing Hooks/Selectors.

### CODE REVIEW

File: client/js/components/statement/assessmentTable/FilterModalSelectItem.vue [hunk 1]
```diff
@@ -9,8 +9,7 @@
 
 <template>
   <div
-    v-if="false === hidden"
-    :data-cy="filterItem.attributes.label">
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/statement/assessmentTable/FilterModalSelectItem.vue:13]: I thing, that we need `data-cy` for `dp-multiselect` not for container, and `filterItem.attributes.label` with space, like `data-cy="Name der Organisation"`
but I can make like
```
<div
    v-if="false === hidden"
    :data-cy="generateDataCy(filterItem.attributes.name, filterItem.attributes)">
```
then resault:
`data-cy="filter_userOrganisation-name-der-organisation"`
it's okay?

[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/statement/assessmentTable/FilterModalSelectItem.vue:13]: Done in https://github.com/demos-europe/demosplan-core/pull/2885/commits/1eb07ce01f51e2775e819c525f95c3774a15885d, with some changes!

[the remark this answers is not in the document]
Comment (salisdemos) [reply client/js/components/statement/assessmentTable/FilterModalSelectItem.vue:13]: I can't test that at the moment, but it looks a bit "too much" for me.
I just want to ensure, that If You remove a data-cy hook, test will break. And we need some sort of communication when we do so, so the testers don't get angry on us

[the remark this answers is not in the document]
Comment (ahmad-demos) [reply client/js/components/statement/assessmentTable/FilterModalSelectItem.vue]: Yes, I understand that, so I will revert it like before!
Done in https://github.com/demos-europe/demosplan-core/pull/2885/commits/b203fe81bef1f2bd97316ce95fe14f5c7a4926c6

**Review by salisdemos — APPROVED** [review]

**Pull request merged**